### PR TITLE
Modified Readme with the latest changes made to the sccli.

### PR DESCRIPTION
**Pull request by Preeticp**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,35 +3,74 @@ adb-utils
...
-  adb-utils uses a systemd service unit file to launch and setup a containerized version of OpenShift. With the systemd unit file, OpenShift can be configured and run in the ADB by issuing the following command: *systemctl start openshift*.
+  adb-utils uses a systemd service unit file to launch and setup a containerized version of OpenShift. With the systemd unit file, OpenShift can be configured and run in the ADB by issuing the following command: ``systemctl start openshift``.
 
 * SCCLI
 
-  SCCLI provides a single point of entry for managing services inside of the ADB. This is primarily of interest to users who use the ADB as a VM they access via *vagrant ssh*.
+  The Service Change CLI (sccli) provides a single point of entry for managing services inside of the ADB. The Kubernetes and Openshift services can not run parallely in ADB because of port conflicts, the sccli provides an easy way to change and run a specific service.
 
-  This CLI can clean the environment and ensure that the requested service is set up properly.
+  This is primarily of interest to users who use the ADB as a VM they access via ``vagrant ssh``.
```
Comment (brgnepal) [inline README.rst]: I think **which** is missing in between "VM" and "they". :

```
This is primarily of interest to users who use the ADB as a VM which they access via ``vagrant ssh``.
```

File: README.rst [hunk 2]
```diff
@@ -3,35 +3,74 @@ adb-utils
...
+     {kubernetes,openshift,docker}
+     kubernetes          start|restart|status|stop (default:start)
+     openshift           start|restart|status|stop (default:start)
+     docker              start|restart|status|stop (default:start)
+
+
+  For instance, to start Kubernetes service, use::
+
+   # sccli kubernetes start
+   # echo $
```
Comment (brgnepal) [inline README.rst]: It should be `echo $?`.

**Comment by praveenkumar** [comment]
LGTM except @budhrg finding.

File: README.rst [hunk 3]
```diff
@@ -3,35 +3,74 @@ adb-utils
...
+
+  If the return code is 0, it implies that the service has behaved (started/restarted/stopped) as expected. If the return status is anything other than 0, it implies that the service has failed to behave as expected.
+
+
+  sccli accepts environment variables to modify how services are configured and deployed.  For the OpenShift service, the following environment variables are recognized:
 
-      usage: sccli [service_name] || [clean]
-      List of possible service_name:
-               k8s openshift
+  ``DOCKER_REGISTRY``, defines which registry the OpenShift containers should be pulled from.  Default value = ``“docker.io"``
```
Comment (brgnepal) [inline README.rst]: `“docker.io"` I think some issue with starting double quote here

File: README.rst [hunk 4]
```diff
@@ -3,35 +3,74 @@ adb-utils
...
 
-  **NOTE:** The image of OpenShift defaults to `docker.io/openshift/origin:v1.1.1`.  If you desire a different version, pass the `DOCKER_REGISTRY`, `IMAGE_NAME` and/or `IMAGE_TAG` environment variables.  For example, to use the `latest` version::
+  ``IMAGE_NAME``, defines which OpenShift image should be used.  Default value = ``“openshift/origin”``
```
Comment (brgnepal) [inline README.rst]: Probably we should use "is" instead of "=" here.

**Comment by brgnepal** [comment]
LGTM with modified changes.

**Comment by praveenkumar** [comment]
LGTM :+1:

**Pull request merged**